diff -NarU5 --exclude=.svn --exclude=CVS --exclude=.git --exclude=_darcs --exclude=.hg osg-client.old/usr/bin/globus-job-run osg-client/usr/bin/globus-job-run
--- osg-client.old/usr/bin/globus-job-run	2015-02-19 00:39:43.000000000 +0000
+++ osg-client/usr/bin/globus-job-run	2015-03-06 00:13:22.000000000 +0000
@@ -19,20 +19,21 @@
 
 #
 # globus-job-run and globus-job-submit (same code base)
 #
 
-prefix="${GLOBUS_LOCATION-/usr}"
-exec_prefix="/usr"
-sbindir="/usr/sbin"
-bindir="/usr/bin"
+root="${OSG_LOCATION-}"
+prefix="${GLOBUS_LOCATION-${root}/usr}"
+exec_prefix="${root}/usr"
+sbindir="${root}/usr/sbin"
+bindir="${root}/usr/bin"
 includedir="${prefix}/include/globus"
 datarootdir="${prefix}/share"
-datadir="/usr/share"
+datadir="${root}/usr/share"
 libexecdir="${datadir}/globus"
-sysconfdir="/etc"
-sharedstatedir="/var/lib"
-localstatedir="/var"
+sysconfdir="${root}/etc"
+sharedstatedir="${root}/var/lib"
+localstatedir="${root}/var"
 
 PATH="${bindir}:${sbindir}:${libexecdir}:${PATH}"
 
 PROGRAM_NAME="${0##*/}"
diff -NarU5 --exclude=.svn --exclude=CVS --exclude=.git --exclude=_darcs --exclude=.hg osg-client.old/usr/bin/globus-job-submit osg-client/usr/bin/globus-job-submit
--- osg-client.old/usr/bin/globus-job-submit	2015-02-19 00:39:43.000000000 +0000
+++ osg-client/usr/bin/globus-job-submit	2015-03-06 00:14:16.000000000 +0000
@@ -19,20 +19,21 @@
 
 #
 # globus-job-run and globus-job-submit (same code base)
 #
 
-prefix="${GLOBUS_LOCATION-/usr}"
-exec_prefix="/usr"
-sbindir="/usr/sbin"
-bindir="/usr/bin"
+root="${OSG_LOCATION-}"
+prefix="${GLOBUS_LOCATION-${root}/usr}"
+exec_prefix="${root}/usr"
+sbindir="${root}/usr/sbin"
+bindir="${root}/usr/bin"
 includedir="${prefix}/include/globus"
 datarootdir="${prefix}/share"
-datadir="/usr/share"
+datadir="${root}/usr/share"
 libexecdir="${datadir}/globus"
-sysconfdir="/etc"
-sharedstatedir="/var/lib"
-localstatedir="/var"
+sysconfdir="${root}/etc"
+sharedstatedir="${root}/var/lib"
+localstatedir="${root}/var"
 
 PATH="${bindir}:${sbindir}:${libexecdir}:${PATH}"
 
 PROGRAM_NAME="${0##*/}"
