diff -Naur --exclude=.svn --exclude=CVS --exclude=.git --exclude=_darcs --exclude=.hg osg-client.old/usr/bin/globus-job-cancel osg-client/usr/bin/globus-job-cancel
--- osg-client.old/usr/bin/globus-job-cancel	2014-01-07 04:43:42.000000000 +0000
+++ osg-client/usr/bin/globus-job-cancel	2014-01-30 20:47:46.682945644 +0000
@@ -16,32 +16,38 @@
 # limitations under the License.
 # 
 
+if test -n "${OSG_LOCATION}" ; then
+    root="${OSG_LOCATION}"
+else
+    root=""
+fi
+
 if test -n "${GLOBUS_LOCATION}" ; then
     prefix="${GLOBUS_LOCATION}"
 else
-    prefix="/usr"
+    prefix="${root}/usr"
 fi
 
-exec_prefix="/usr"
-sbindir="/usr/sbin"
-bindir="/usr/bin"
+exec_prefix="${root}/usr"
+sbindir="${root}/usr/sbin"
+bindir="${root}/usr/bin"
 
 case `uname -m` in
     aarch64|ppc64|s390x|sparc64|x86_64)
-        libdir="/usr/lib64"
+        libdir="${root}/usr/lib64"
         ;;
     *)
-        libdir="/usr/lib"
+        libdir="${root}/usr/lib"
         ;;
 esac
 
 includedir="${prefix}/include/globus"
 datarootdir="${prefix}/share"
-datadir="/usr/share"
+datadir="${root}/usr/share"
 libexecdir="${datadir}/globus"
-sysconfdir="/etc"
-sharedstatedir="/var/lib"
-localstatedir="/var"
+sysconfdir="${root}/etc"
+sharedstatedir="${root}/var/lib"
+localstatedir="${root}/var"
 aclocaldir="${datadir}/globus/aclocal"
 
 . ${libexecdir}/globus-script-initializer
diff -Naur --exclude=.svn --exclude=CVS --exclude=.git --exclude=_darcs --exclude=.hg osg-client.old/usr/bin/globus-job-clean osg-client/usr/bin/globus-job-clean
--- osg-client.old/usr/bin/globus-job-clean	2014-01-07 04:43:42.000000000 +0000
+++ osg-client/usr/bin/globus-job-clean	2014-01-30 20:49:09.732955732 +0000
@@ -16,32 +16,38 @@
 # limitations under the License.
 # 
 
+if test -n "${OSG_LOCATION}" ; then
+    root="${OSG_LOCATION}"
+else
+    root=""
+fi
+
 if test -n "${GLOBUS_LOCATION}" ; then
     prefix="${GLOBUS_LOCATION}"
 else
-    prefix="/usr"
+    prefix="${root}/usr"
 fi
 
-exec_prefix="/usr"
-sbindir="/usr/sbin"
-bindir="/usr/bin"
+exec_prefix="${root}/usr"
+sbindir="${root}/usr/sbin"
+bindir="${root}/usr/bin"
 
 case `uname -m` in
     aarch64|ppc64|s390x|sparc64|x86_64)
-        libdir="/usr/lib64"
+        libdir="${root}/usr/lib64"
         ;;
     *)
-        libdir="/usr/lib"
+        libdir="${root}/usr/lib"
         ;;
 esac
 
 includedir="${prefix}/include/globus"
 datarootdir="${prefix}/share"
-datadir="/usr/share"
+datadir="${root}/usr/share"
 libexecdir="${datadir}/globus"
-sysconfdir="/etc"
-sharedstatedir="/var/lib"
-localstatedir="/var"
+sysconfdir="${root}/etc"
+sharedstatedir="${root}/var/lib"
+localstatedir="${root}/var"
 aclocaldir="${datadir}/globus/aclocal"
 
 . ${libexecdir}/globus-script-initializer
diff -Naur --exclude=.svn --exclude=CVS --exclude=.git --exclude=_darcs --exclude=.hg osg-client.old/usr/bin/globus-job-get-output osg-client/usr/bin/globus-job-get-output
--- osg-client.old/usr/bin/globus-job-get-output	2014-01-07 04:43:42.000000000 +0000
+++ osg-client/usr/bin/globus-job-get-output	2014-01-30 20:50:26.621963170 +0000
@@ -16,32 +16,38 @@
 # limitations under the License.
 # 
 
+if test -n "${OSG_LOCATION}" ; then
+    root="${OSG_LOCATION}"
+else
+    root=""
+fi
+
 if test -n "${GLOBUS_LOCATION}" ; then
     prefix="${GLOBUS_LOCATION}"
 else
-    prefix="/usr"
+    prefix="${root}/usr"
 fi
 
-exec_prefix="/usr"
-sbindir="/usr/sbin"
-bindir="/usr/bin"
+exec_prefix="${root}/usr"
+sbindir="${root}/usr/sbin"
+bindir="${root}/usr/bin"
 
 case `uname -m` in
     aarch64|ppc64|s390x|sparc64|x86_64)
-        libdir="/usr/lib64"
+        libdir="${root}/usr/lib64"
         ;;
     *)
-        libdir="/usr/lib"
+        libdir="${root}/usr/lib"
         ;;
 esac
 
 includedir="${prefix}/include/globus"
 datarootdir="${prefix}/share"
-datadir="/usr/share"
+datadir="${root}/usr/share"
 libexecdir="${datadir}/globus"
-sysconfdir="/etc"
-sharedstatedir="/var/lib"
-localstatedir="/var"
+sysconfdir="${root}/etc"
+sharedstatedir="${root}/var/lib"
+localstatedir="${root}/var"
 aclocaldir="${datadir}/globus/aclocal"
 
 . ${libexecdir}/globus-script-initializer
diff -Naur --exclude=.svn --exclude=CVS --exclude=.git --exclude=_darcs --exclude=.hg osg-client.old/usr/bin/globus-job-status osg-client/usr/bin/globus-job-status
--- osg-client.old/usr/bin/globus-job-status	2014-01-07 04:43:42.000000000 +0000
+++ osg-client/usr/bin/globus-job-status	2014-01-30 20:50:51.702967750 +0000
@@ -16,32 +16,38 @@
 # limitations under the License.
 # 
 
+if test -n "${OSG_LOCATION}" ; then
+    root="${OSG_LOCATION}"
+else
+    root=""
+fi
+
 if test -n "${GLOBUS_LOCATION}" ; then
     prefix="${GLOBUS_LOCATION}"
 else
-    prefix="/usr"
+    prefix="${root}/usr"
 fi
 
-exec_prefix="/usr"
-sbindir="/usr/sbin"
-bindir="/usr/bin"
+exec_prefix="${root}/usr"
+sbindir="${root}/usr/sbin"
+bindir="${root}/usr/bin"
 
 case `uname -m` in
     aarch64|ppc64|s390x|sparc64|x86_64)
-        libdir="/usr/lib64"
+        libdir="${root}/usr/lib64"
         ;;
     *)
-        libdir="/usr/lib"
+        libdir="${root}/usr/lib"
         ;;
 esac
 
 includedir="${prefix}/include/globus"
 datarootdir="${prefix}/share"
-datadir="/usr/share"
+datadir="${root}/usr/share"
 libexecdir="${datadir}/globus"
-sysconfdir="/etc"
-sharedstatedir="/var/lib"
-localstatedir="/var"
+sysconfdir="${root}/etc"
+sharedstatedir="${root}/var/lib"
+localstatedir="${root}/var"
 aclocaldir="${datadir}/globus/aclocal"
 
 . ${libexecdir}/globus-script-initializer
