diff -Naur --exclude=.svn --exclude=CVS --exclude=.git --exclude=_darcs --exclude=.hg osg-client.orig/usr/bin/grid-cert-info osg-client/usr/bin/grid-cert-info
--- osg-client.orig/usr/bin/grid-cert-info	2012-10-26 22:21:46.000000000 +0000
+++ osg-client/usr/bin/grid-cert-info	2013-05-03 16:57:04.187809000 +0000
@@ -22,37 +22,43 @@
 # Easily extract information from a user's cert.
 #
 
+if test -n "${OSG_LOCATION}" ; then
+    root="${OSG_LOCATION}"
+else
+    root=""
+fi
 
 if test -n "${GLOBUS_LOCATION}" ; then
     prefix="${GLOBUS_LOCATION}"
 else
-    prefix="/usr"
+    prefix="${root}/usr"
 fi
 
-exec_prefix="/usr"
-sbindir="/usr/sbin"
-bindir="/usr/bin"
+exec_prefix="${root}/usr"
+sbindir="${root}/usr/sbin"
+bindir="${root}/usr/bin"
 
 case `uname -m` in
     ppc64|s390x|sparc64|x86_64)
-        libdir="/usr/lib64"
+        libdir="${root}/usr/lib64"
         ;;
     *)
-        libdir="/usr/lib"
+        libdir="${root}/usr/lib"
         ;;
 esac
 
 includedir="${prefix}/include/globus"
 datarootdir="${prefix}/share"
-datadir="/usr/share"
+datadir="${root}/usr/share"
 libexecdir="${datadir}/globus"
-sysconfdir="/etc"
-sharedstatedir="/usr/com"
-localstatedir="/var"
+sysconfdir="${root}/etc"
+sharedstatedir="${root}/usr/com"
+localstatedir="${root}/var"
 aclocaldir="${datadir}/globus/aclocal"
 
 . ${libexecdir}/globus-script-initializer
 
+
 globus_source ${libexecdir}/globus-sh-tools.sh
 
 PROGRAM_NAME=`echo $0 | ${GLOBUS_SH_SED-sed} 's|.*/||g'`
diff -Naur --exclude=.svn --exclude=CVS --exclude=.git --exclude=_darcs --exclude=.hg osg-client.orig/usr/bin/grid-cert-request osg-client/usr/bin/grid-cert-request
--- osg-client.orig/usr/bin/grid-cert-request	2012-10-26 22:21:46.000000000 +0000
+++ osg-client/usr/bin/grid-cert-request	2013-05-03 16:58:15.500887000 +0000
@@ -39,33 +39,38 @@
 
 
 
+if test -n "${OSG_LOCATION}" ; then
+    root="${OSG_LOCATION}"
+else
+    root=""
+fi
 
 if test -n "${GLOBUS_LOCATION}" ; then
     prefix="${GLOBUS_LOCATION}"
 else
-    prefix="/usr"
+    prefix="${root}/usr"
 fi
 
-exec_prefix="/usr"
-sbindir="/usr/sbin"
-bindir="/usr/bin"
+exec_prefix="${root}/usr"
+sbindir="${root}/usr/sbin"
+bindir="${root}/usr/bin"
 
 case `uname -m` in
     ppc64|s390x|sparc64|x86_64)
-        libdir="/usr/lib64"
+        libdir="${root}/usr/lib64"
         ;;
     *)
-        libdir="/usr/lib"
+        libdir="${root}/usr/lib"
         ;;
 esac
 
 includedir="${prefix}/include/globus"
 datarootdir="${prefix}/share"
-datadir="/usr/share"
+datadir="${root}/usr/share"
 libexecdir="${datadir}/globus"
-sysconfdir="/etc"
-sharedstatedir="/usr/com"
-localstatedir="/var"
+sysconfdir="${root}/etc"
+sharedstatedir="${root}/usr/com"
+localstatedir="${root}/var"
 aclocaldir="${datadir}/globus/aclocal"
 
 . ${libexecdir}/globus-script-initializer
diff -Naur --exclude=.svn --exclude=CVS --exclude=.git --exclude=_darcs --exclude=.hg osg-client.orig/usr/bin/grid-change-pass-phrase osg-client/usr/bin/grid-change-pass-phrase
--- osg-client.orig/usr/bin/grid-change-pass-phrase	2012-10-26 22:21:46.000000000 +0000
+++ osg-client/usr/bin/grid-change-pass-phrase	2013-05-03 16:59:09.906472000 +0000
@@ -21,32 +21,38 @@
 #
 
 
+if test -n "${OSG_LOCATION}" ; then
+    root="${OSG_LOCATION}"
+else
+    root=""
+fi
+
 if test -n "${GLOBUS_LOCATION}" ; then
     prefix="${GLOBUS_LOCATION}"
 else
-    prefix="/usr"
+    prefix="${root}/usr"
 fi
 
-exec_prefix="/usr"
-sbindir="/usr/sbin"
-bindir="/usr/bin"
+exec_prefix="${root}/usr"
+sbindir="${root}/usr/sbin"
+bindir="${root}/usr/bin"
 
 case `uname -m` in
     ppc64|s390x|sparc64|x86_64)
-        libdir="/usr/lib64"
+        libdir="${root}/usr/lib64"
         ;;
     *)
-        libdir="/usr/lib"
+        libdir="${root}/usr/lib"
         ;;
 esac
 
 includedir="${prefix}/include/globus"
 datarootdir="${prefix}/share"
-datadir="/usr/share"
+datadir="${root}/usr/share"
 libexecdir="${datadir}/globus"
-sysconfdir="/etc"
-sharedstatedir="/usr/com"
-localstatedir="/var"
+sysconfdir="${root}/etc"
+sharedstatedir="${root}/usr/com"
+localstatedir="${root}/var"
 aclocaldir="${datadir}/globus/aclocal"
 
 . ${libexecdir}/globus-script-initializer
