diff -Naur --exclude=.svn --exclude=CVS --exclude=.git --exclude=_darcs --exclude=.hg osg-wn-client.old/usr/sbin/fetch-crl3 osg-wn-client/usr/sbin/fetch-crl3
--- osg-wn-client.old/usr/sbin/fetch-crl3	2013-07-23 13:35:39.000000000 +0000
+++ osg-wn-client/usr/sbin/fetch-crl3	2013-09-09 23:06:00.350813025 +0000
@@ -151,8 +151,10 @@
   $help and &help and exit(0);
   $showversion and &showversion and exit(0);
 
-  $configfile ||= ( -e "/etc/fetch-crl3.conf" and "/etc/fetch-crl3.conf" );
-  $configfile ||= ( -e "/etc/fetch-crl3.cnf" and "/etc/fetch-crl3.cnf" );
+  my $osg_location = $ENV{OSG_LOCATION} || "";
+
+  $configfile ||= ( -e "${osg_location}/etc/fetch-crl3.conf" and "${osg_location}/etc/fetch-crl3.conf" );
+  $configfile ||= ( -e "${osg_location}/etc/fetch-crl3.cnf" and "${osg_location}/etc/fetch-crl3.cnf" );
 
   $cnf = ConfigTiny->new();
   $configfile and 
