diff -Naur --exclude=.svn --exclude=CVS --exclude=.git --exclude=_darcs --exclude=.hg osg-wn-client.old/usr/sbin/fetch-crl osg-wn-client/usr/sbin/fetch-crl
--- osg-wn-client.old/usr/sbin/fetch-crl	2013-07-23 13:15:23.000000000 +0000
+++ osg-wn-client/usr/sbin/fetch-crl	2013-09-09 23:13:19.430469427 +0000
@@ -151,8 +151,10 @@
   $help and &help and exit(0);
   $showversion and &showversion and exit(0);
 
-  $configfile ||= ( -e "/etc/fetch-crl.conf" and "/etc/fetch-crl.conf" );
-  $configfile ||= ( -e "/etc/fetch-crl.cnf" and "/etc/fetch-crl.cnf" );
+  my ($osg_location) = $ENV{OSG_LOCATION} || "";
+
+  $configfile ||= ( -e "${osg_location}/etc/fetch-crl.conf" and "${osg_location}/etc/fetch-crl.conf" );
+  $configfile ||= ( -e "${osg_location}/etc/fetch-crl.cnf" and "${osg_location}/etc/fetch-crl.cnf" );
 
   $cnf = ConfigTiny->new();
   $configfile and 
