diff -NarU5 --exclude=.svn --exclude=CVS --exclude=.git --exclude=_darcs --exclude=.hg osg-wn-client.old/usr/lib/perl5/vendor_perl/5.8.8/Globus/Core/Paths.pm osg-wn-client/usr/lib/perl5/vendor_perl/5.8.8/Globus/Core/Paths.pm
--- osg-wn-client.old/usr/lib/perl5/vendor_perl/5.8.8/Globus/Core/Paths.pm	2015-02-19 04:38:21.000000000 +0000
+++ osg-wn-client/usr/lib/perl5/vendor_perl/5.8.8/Globus/Core/Paths.pm	2015-03-05 23:30:29.000000000 +0000
@@ -45,27 +45,35 @@
         $last = $path;
     }
     return $path;
 }
 
+if (exists $ENV{OSG_LOCATION})
+{
+    $root = $ENV{OSG_LOCATION};
+}
+else
+{
+    $root = '';
+}
 if (exists $ENV{GLOBUS_LOCATION})
 {
     $prefix = $ENV{GLOBUS_LOCATION};
 }
 else
 {
-    $prefix = "/usr";
+    $prefix = "${root}/usr";
 }
-$exec_prefix = "/usr";
-$sbindir = "/usr/sbin";
-$bindir = "/usr/bin";
+$exec_prefix = "${root}/usr";
+$sbindir = "${root}/usr/sbin";
+$bindir = "${root}/usr/bin";
 $includedir = "${prefix}/include/globus";
 $datarootdir = "${prefix}/share";
-$datadir = "/usr/share";
-$mandir = "/usr/share/man";
+$datadir = "${root}/usr/share";
+$mandir = "${root}/usr/share/man";
 $libexecdir = "${datadir}/globus";
-$sysconfdir = "/etc";
-$sharedstatedir = "/usr/com";
-$localstatedir = "/var";
-$perlmoduledir = "/usr/lib/perl5/vendor_perl/5.8.8";
+$sysconfdir = "${root}/etc";
+$sharedstatedir = "${root}/usr/com";
+$localstatedir = "${root}/var";
+$perlmoduledir = "${root}/usr/lib/perl5/vendor_perl/5.8.8";
 
 1;
