diff -Naur --exclude=.svn --exclude=CVS --exclude=.git --exclude=_darcs --exclude=.hg osg-wn-client.old/etc/sysconfig/dcache-srmclient osg-wn-client/etc/sysconfig/dcache-srmclient
--- osg-wn-client.old/etc/sysconfig/dcache-srmclient	2013-06-08 04:24:41.000000000 +0000
+++ osg-wn-client/etc/sysconfig/dcache-srmclient	2013-08-12 23:53:06.124406651 +0000
@@ -1,2 +1,2 @@
-export SRM_PATH=/usr/share/srm
-export SRM_CONFIG=/etc/dcache-srmclient-config.xml
+export SRM_PATH=${OSG_LOCATION-}/usr/share/srm
+export SRM_CONFIG=${OSG_LOCATION-}/etc/dcache-srmclient-config.xml
