diff -Naur --exclude=.svn --exclude=CVS --exclude=.git --exclude=_darcs --exclude=.hg osg-wn-client.old/usr/bin/srm-abort-files osg-wn-client/usr/bin/srm-abort-files
--- osg-wn-client.old/usr/bin/srm-abort-files	2013-06-08 04:24:41.000000000 +0000
+++ osg-wn-client/usr/bin/srm-abort-files	2013-08-12 23:30:07.448688121 +0000
@@ -1,7 +1,7 @@
 #!/bin/sh
 
-if [ -e /etc/sysconfig/dcache-srmclient ]; then
-    . /etc/sysconfig/dcache-srmclient
+if [ -e ${OSG_LOCATION-}/etc/sysconfig/dcache-srmclient ]; then
+    . ${OSG_LOCATION-}/etc/sysconfig/dcache-srmclient
 fi
 
 if [ -z "$SRM_PATH" ]; then
diff -Naur --exclude=.svn --exclude=CVS --exclude=.git --exclude=_darcs --exclude=.hg osg-wn-client.old/usr/bin/srm-abort-request osg-wn-client/usr/bin/srm-abort-request
--- osg-wn-client.old/usr/bin/srm-abort-request	2013-06-08 04:24:41.000000000 +0000
+++ osg-wn-client/usr/bin/srm-abort-request	2013-08-12 23:30:19.290689495 +0000
@@ -1,7 +1,7 @@
 #!/bin/sh
 
-if [ -e /etc/sysconfig/dcache-srmclient ]; then
-    . /etc/sysconfig/dcache-srmclient
+if [ -e ${OSG_LOCATION-}/etc/sysconfig/dcache-srmclient ]; then
+    . ${OSG_LOCATION-}/etc/sysconfig/dcache-srmclient
 fi
 
 if [ -z "$SRM_PATH" ]; then
diff -Naur --exclude=.svn --exclude=CVS --exclude=.git --exclude=_darcs --exclude=.hg osg-wn-client.old/usr/bin/srm-advisory-delete osg-wn-client/usr/bin/srm-advisory-delete
--- osg-wn-client.old/usr/bin/srm-advisory-delete	2013-06-08 04:24:41.000000000 +0000
+++ osg-wn-client/usr/bin/srm-advisory-delete	2013-08-12 23:31:18.338681553 +0000
@@ -1,7 +1,7 @@
 #!/bin/sh
 
-if [ -e /etc/sysconfig/dcache-srmclient ]; then
-    . /etc/sysconfig/dcache-srmclient
+if [ -e ${OSG_LOCATION-}/etc/sysconfig/dcache-srmclient ]; then
+    . ${OSG_LOCATION-}/etc/sysconfig/dcache-srmclient
 fi
 
 if [ -z "$SRM_PATH" ]; then
diff -Naur --exclude=.svn --exclude=CVS --exclude=.git --exclude=_darcs --exclude=.hg osg-wn-client.old/usr/bin/srm-bring-online osg-wn-client/usr/bin/srm-bring-online
--- osg-wn-client.old/usr/bin/srm-bring-online	2013-06-08 04:24:41.000000000 +0000
+++ osg-wn-client/usr/bin/srm-bring-online	2013-08-12 23:37:13.519560396 +0000
@@ -1,7 +1,7 @@
 #!/bin/sh
 
-if [ -e /etc/sysconfig/dcache-srmclient ]; then
-    . /etc/sysconfig/dcache-srmclient
+if [ -e ${OSG_LOCATION-}/etc/sysconfig/dcache-srmclient ]; then
+    . ${OSG_LOCATION-}/etc/sysconfig/dcache-srmclient
 fi
 
 if [ -z "$SRM_PATH" ]; then
diff -Naur --exclude=.svn --exclude=CVS --exclude=.git --exclude=_darcs --exclude=.hg osg-wn-client.old/usr/bin/srm-check-permissions osg-wn-client/usr/bin/srm-check-permissions
--- osg-wn-client.old/usr/bin/srm-check-permissions	2013-06-08 04:24:41.000000000 +0000
+++ osg-wn-client/usr/bin/srm-check-permissions	2013-08-12 23:38:05.638550929 +0000
@@ -1,7 +1,7 @@
 #!/bin/sh
 
-if [ -e /etc/sysconfig/dcache-srmclient ]; then
-    . /etc/sysconfig/dcache-srmclient
+if [ -e ${OSG_LOCATION-}/etc/sysconfig/dcache-srmclient ]; then
+    . ${OSG_LOCATION-}/etc/sysconfig/dcache-srmclient
 fi
 
 if [ -z "$SRM_PATH" ]; then
diff -Naur --exclude=.svn --exclude=CVS --exclude=.git --exclude=_darcs --exclude=.hg osg-wn-client.old/usr/bin/srm-extend-file-lifetime osg-wn-client/usr/bin/srm-extend-file-lifetime
--- osg-wn-client.old/usr/bin/srm-extend-file-lifetime	2013-06-08 04:24:41.000000000 +0000
+++ osg-wn-client/usr/bin/srm-extend-file-lifetime	2013-08-12 23:38:16.872545264 +0000
@@ -1,7 +1,7 @@
 #!/bin/sh
 
-if [ -e /etc/sysconfig/dcache-srmclient ]; then
-    . /etc/sysconfig/dcache-srmclient
+if [ -e ${OSG_LOCATION-}/etc/sysconfig/dcache-srmclient ]; then
+    . ${OSG_LOCATION-}/etc/sysconfig/dcache-srmclient
 fi
 
 if [ -z "$SRM_PATH" ]; then
diff -Naur --exclude=.svn --exclude=CVS --exclude=.git --exclude=_darcs --exclude=.hg osg-wn-client.old/usr/bin/srm-get-metadata osg-wn-client/usr/bin/srm-get-metadata
--- osg-wn-client.old/usr/bin/srm-get-metadata	2013-06-08 04:24:41.000000000 +0000
+++ osg-wn-client/usr/bin/srm-get-metadata	2013-08-12 23:38:42.350542083 +0000
@@ -1,7 +1,7 @@
 #!/bin/sh
 
-if [ -e /etc/sysconfig/dcache-srmclient ]; then
-    . /etc/sysconfig/dcache-srmclient
+if [ -e ${OSG_LOCATION-}/etc/sysconfig/dcache-srmclient ]; then
+    . ${OSG_LOCATION-}/etc/sysconfig/dcache-srmclient
 fi
 
 if [ -z "$SRM_PATH" ]; then
diff -Naur --exclude=.svn --exclude=CVS --exclude=.git --exclude=_darcs --exclude=.hg osg-wn-client.old/usr/bin/srm-get-permissions osg-wn-client/usr/bin/srm-get-permissions
--- osg-wn-client.old/usr/bin/srm-get-permissions	2013-06-08 04:24:41.000000000 +0000
+++ osg-wn-client/usr/bin/srm-get-permissions	2013-08-12 23:38:55.023538408 +0000
@@ -1,7 +1,7 @@
 #!/bin/sh
 
-if [ -e /etc/sysconfig/dcache-srmclient ]; then
-    . /etc/sysconfig/dcache-srmclient
+if [ -e ${OSG_LOCATION-}/etc/sysconfig/dcache-srmclient ]; then
+    . ${OSG_LOCATION-}/etc/sysconfig/dcache-srmclient
 fi
 
 if [ -z "$SRM_PATH" ]; then
diff -Naur --exclude=.svn --exclude=CVS --exclude=.git --exclude=_darcs --exclude=.hg osg-wn-client.old/usr/bin/srm-get-request-status osg-wn-client/usr/bin/srm-get-request-status
--- osg-wn-client.old/usr/bin/srm-get-request-status	2013-06-08 04:24:41.000000000 +0000
+++ osg-wn-client/usr/bin/srm-get-request-status	2013-08-12 23:39:56.804519859 +0000
@@ -1,7 +1,7 @@
 #!/bin/sh
 
-if [ -e /etc/sysconfig/dcache-srmclient ]; then
-    . /etc/sysconfig/dcache-srmclient
+if [ -e ${OSG_LOCATION-}/etc/sysconfig/dcache-srmclient ]; then
+    . ${OSG_LOCATION-}/etc/sysconfig/dcache-srmclient
 fi
 
 if [ -z "$SRM_PATH" ]; then
diff -Naur --exclude=.svn --exclude=CVS --exclude=.git --exclude=_darcs --exclude=.hg osg-wn-client.old/usr/bin/srm-get-request-summary osg-wn-client/usr/bin/srm-get-request-summary
--- osg-wn-client.old/usr/bin/srm-get-request-summary	2013-06-08 04:24:41.000000000 +0000
+++ osg-wn-client/usr/bin/srm-get-request-summary	2013-08-12 23:40:21.492521411 +0000
@@ -1,7 +1,7 @@
 #!/bin/sh
 
-if [ -e /etc/sysconfig/dcache-srmclient ]; then
-    . /etc/sysconfig/dcache-srmclient
+if [ -e ${OSG_LOCATION-}/etc/sysconfig/dcache-srmclient ]; then
+    . ${OSG_LOCATION-}/etc/sysconfig/dcache-srmclient
 fi
 
 if [ -z "$SRM_PATH" ]; then
diff -Naur --exclude=.svn --exclude=CVS --exclude=.git --exclude=_darcs --exclude=.hg osg-wn-client.old/usr/bin/srm-get-request-tokens osg-wn-client/usr/bin/srm-get-request-tokens
--- osg-wn-client.old/usr/bin/srm-get-request-tokens	2013-06-08 04:24:41.000000000 +0000
+++ osg-wn-client/usr/bin/srm-get-request-tokens	2013-08-12 23:40:27.048516428 +0000
@@ -1,7 +1,7 @@
 #!/bin/sh
 
-if [ -e /etc/sysconfig/dcache-srmclient ]; then
-    . /etc/sysconfig/dcache-srmclient
+if [ -e ${OSG_LOCATION-}/etc/sysconfig/dcache-srmclient ]; then
+    . ${OSG_LOCATION-}/etc/sysconfig/dcache-srmclient
 fi
 
 if [ -z "$SRM_PATH" ]; then
diff -Naur --exclude=.svn --exclude=CVS --exclude=.git --exclude=_darcs --exclude=.hg osg-wn-client.old/usr/bin/srm-get-space-metadata osg-wn-client/usr/bin/srm-get-space-metadata
--- osg-wn-client.old/usr/bin/srm-get-space-metadata	2013-06-08 04:24:41.000000000 +0000
+++ osg-wn-client/usr/bin/srm-get-space-metadata	2013-08-12 23:40:43.018526250 +0000
@@ -1,7 +1,7 @@
 #!/bin/sh
 
-if [ -e /etc/sysconfig/dcache-srmclient ]; then
-    . /etc/sysconfig/dcache-srmclient
+if [ -e ${OSG_LOCATION-}/etc/sysconfig/dcache-srmclient ]; then
+    . ${OSG_LOCATION-}/etc/sysconfig/dcache-srmclient
 fi
 
 if [ -z "$SRM_PATH" ]; then
diff -Naur --exclude=.svn --exclude=CVS --exclude=.git --exclude=_darcs --exclude=.hg osg-wn-client.old/usr/bin/srm-get-space-tokens osg-wn-client/usr/bin/srm-get-space-tokens
--- osg-wn-client.old/usr/bin/srm-get-space-tokens	2013-06-08 04:24:41.000000000 +0000
+++ osg-wn-client/usr/bin/srm-get-space-tokens	2013-08-12 23:40:48.674512304 +0000
@@ -1,7 +1,7 @@
 #!/bin/sh
 
-if [ -e /etc/sysconfig/dcache-srmclient ]; then
-    . /etc/sysconfig/dcache-srmclient
+if [ -e ${OSG_LOCATION-}/etc/sysconfig/dcache-srmclient ]; then
+    . ${OSG_LOCATION-}/etc/sysconfig/dcache-srmclient
 fi
 
 if [ -z "$SRM_PATH" ]; then
diff -Naur --exclude=.svn --exclude=CVS --exclude=.git --exclude=_darcs --exclude=.hg osg-wn-client.old/usr/bin/srm-release-files osg-wn-client/usr/bin/srm-release-files
--- osg-wn-client.old/usr/bin/srm-release-files	2013-06-08 04:24:41.000000000 +0000
+++ osg-wn-client/usr/bin/srm-release-files	2013-08-12 23:41:01.762521676 +0000
@@ -1,7 +1,7 @@
 #!/bin/sh
 
-if [ -e /etc/sysconfig/dcache-srmclient ]; then
-    . /etc/sysconfig/dcache-srmclient
+if [ -e ${OSG_LOCATION-}/etc/sysconfig/dcache-srmclient ]; then
+    . ${OSG_LOCATION-}/etc/sysconfig/dcache-srmclient
 fi
 
 if [ -z "$SRM_PATH" ]; then
diff -Naur --exclude=.svn --exclude=CVS --exclude=.git --exclude=_darcs --exclude=.hg osg-wn-client.old/usr/bin/srm-release-space osg-wn-client/usr/bin/srm-release-space
--- osg-wn-client.old/usr/bin/srm-release-space	2013-06-08 04:24:41.000000000 +0000
+++ osg-wn-client/usr/bin/srm-release-space	2013-08-12 23:41:12.724526377 +0000
@@ -1,7 +1,7 @@
 #!/bin/sh
 
-if [ -e /etc/sysconfig/dcache-srmclient ]; then
-    . /etc/sysconfig/dcache-srmclient
+if [ -e ${OSG_LOCATION-}/etc/sysconfig/dcache-srmclient ]; then
+    . ${OSG_LOCATION-}/etc/sysconfig/dcache-srmclient
 fi
 
 if [ -z "$SRM_PATH" ]; then
diff -Naur --exclude=.svn --exclude=CVS --exclude=.git --exclude=_darcs --exclude=.hg osg-wn-client.old/usr/bin/srm-reserve-space osg-wn-client/usr/bin/srm-reserve-space
--- osg-wn-client.old/usr/bin/srm-reserve-space	2013-06-08 04:24:41.000000000 +0000
+++ osg-wn-client/usr/bin/srm-reserve-space	2013-08-12 23:41:37.139502273 +0000
@@ -1,7 +1,7 @@
 #!/bin/sh
 
 if [ -z "$SRM_PATH" ]; then
-    SRM_PATH="/usr/share/srm"
+    SRM_PATH="${OSG_LOCATION-}/usr/share/srm"
 fi
 
 if [ ! -d "$SRM_PATH" ]; then
diff -Naur --exclude=.svn --exclude=CVS --exclude=.git --exclude=_darcs --exclude=.hg osg-wn-client.old/usr/bin/srm-set-permissions osg-wn-client/usr/bin/srm-set-permissions
--- osg-wn-client.old/usr/bin/srm-set-permissions	2013-06-08 04:24:41.000000000 +0000
+++ osg-wn-client/usr/bin/srm-set-permissions	2013-08-12 23:41:51.573505312 +0000
@@ -1,7 +1,7 @@
 #!/bin/sh
 
-if [ -e /etc/sysconfig/dcache-srmclient ]; then
-    . /etc/sysconfig/dcache-srmclient
+if [ -e ${OSG_LOCATION-}/etc/sysconfig/dcache-srmclient ]; then
+    . ${OSG_LOCATION-}/etc/sysconfig/dcache-srmclient
 fi
 
 if [ -z "$SRM_PATH" ]; then
diff -Naur --exclude=.svn --exclude=CVS --exclude=.git --exclude=_darcs --exclude=.hg osg-wn-client.old/usr/bin/srm-storage-element-info osg-wn-client/usr/bin/srm-storage-element-info
--- osg-wn-client.old/usr/bin/srm-storage-element-info	2013-06-08 04:24:41.000000000 +0000
+++ osg-wn-client/usr/bin/srm-storage-element-info	2013-08-12 23:42:14.627505116 +0000
@@ -1,7 +1,7 @@
 #!/bin/sh
 
-if [ -e /etc/sysconfig/dcache-srmclient ]; then
-    . /etc/sysconfig/dcache-srmclient
+if [ -e ${OSG_LOCATION-}/etc/sysconfig/dcache-srmclient ]; then
+    . ${OSG_LOCATION-}/etc/sysconfig/dcache-srmclient
 fi
 
 if [ -z "$SRM_PATH" ]; then
diff -Naur --exclude=.svn --exclude=CVS --exclude=.git --exclude=_darcs --exclude=.hg osg-wn-client.old/usr/bin/srmcp osg-wn-client/usr/bin/srmcp
--- osg-wn-client.old/usr/bin/srmcp	2013-06-08 04:24:41.000000000 +0000
+++ osg-wn-client/usr/bin/srmcp	2013-08-12 23:42:26.052500912 +0000
@@ -1,7 +1,7 @@
 #!/bin/sh
 
-if [ -e /etc/sysconfig/dcache-srmclient ]; then
-    . /etc/sysconfig/dcache-srmclient
+if [ -e ${OSG_LOCATION-}/etc/sysconfig/dcache-srmclient ]; then
+    . ${OSG_LOCATION-}/etc/sysconfig/dcache-srmclient
 fi
 
 if [ -z "$SRM_PATH" ]; then
diff -Naur --exclude=.svn --exclude=CVS --exclude=.git --exclude=_darcs --exclude=.hg osg-wn-client.old/usr/bin/srmls osg-wn-client/usr/bin/srmls
--- osg-wn-client.old/usr/bin/srmls	2013-06-08 04:24:41.000000000 +0000
+++ osg-wn-client/usr/bin/srmls	2013-08-12 23:42:32.372499925 +0000
@@ -1,7 +1,7 @@
 #!/bin/sh
 
-if [ -e /etc/sysconfig/dcache-srmclient ]; then
-    . /etc/sysconfig/dcache-srmclient
+if [ -e ${OSG_LOCATION-}/etc/sysconfig/dcache-srmclient ]; then
+    . ${OSG_LOCATION-}/etc/sysconfig/dcache-srmclient
 fi
 
 if [ -z "$SRM_PATH" ]; then
diff -Naur --exclude=.svn --exclude=CVS --exclude=.git --exclude=_darcs --exclude=.hg osg-wn-client.old/usr/bin/srmmkdir osg-wn-client/usr/bin/srmmkdir
--- osg-wn-client.old/usr/bin/srmmkdir	2013-06-08 04:24:41.000000000 +0000
+++ osg-wn-client/usr/bin/srmmkdir	2013-08-12 23:42:38.173498392 +0000
@@ -1,7 +1,7 @@
 #!/bin/sh
 
-if [ -e /etc/sysconfig/dcache-srmclient ]; then
-    . /etc/sysconfig/dcache-srmclient
+if [ -e ${OSG_LOCATION-}/etc/sysconfig/dcache-srmclient ]; then
+    . ${OSG_LOCATION-}/etc/sysconfig/dcache-srmclient
 fi
 
 if [ -z "$SRM_PATH" ]; then
diff -Naur --exclude=.svn --exclude=CVS --exclude=.git --exclude=_darcs --exclude=.hg osg-wn-client.old/usr/bin/srmmv osg-wn-client/usr/bin/srmmv
--- osg-wn-client.old/usr/bin/srmmv	2013-06-08 04:24:41.000000000 +0000
+++ osg-wn-client/usr/bin/srmmv	2013-08-12 23:42:43.380500697 +0000
@@ -1,7 +1,7 @@
 #!/bin/sh
 
-if [ -e /etc/sysconfig/dcache-srmclient ]; then
-    . /etc/sysconfig/dcache-srmclient
+if [ -e ${OSG_LOCATION-}/etc/sysconfig/dcache-srmclient ]; then
+    . ${OSG_LOCATION-}/etc/sysconfig/dcache-srmclient
 fi
 
 if [ -z "$SRM_PATH" ]; then
diff -Naur --exclude=.svn --exclude=CVS --exclude=.git --exclude=_darcs --exclude=.hg osg-wn-client.old/usr/bin/srmping osg-wn-client/usr/bin/srmping
--- osg-wn-client.old/usr/bin/srmping	2013-06-08 04:24:41.000000000 +0000
+++ osg-wn-client/usr/bin/srmping	2013-08-12 23:42:48.660496927 +0000
@@ -1,7 +1,7 @@
 #!/bin/sh
 
-if [ -e /etc/sysconfig/dcache-srmclient ]; then
-    . /etc/sysconfig/dcache-srmclient
+if [ -e ${OSG_LOCATION-}/etc/sysconfig/dcache-srmclient ]; then
+    . ${OSG_LOCATION-}/etc/sysconfig/dcache-srmclient
 fi
 
 if [ -z "$SRM_PATH" ]; then
diff -Naur --exclude=.svn --exclude=CVS --exclude=.git --exclude=_darcs --exclude=.hg osg-wn-client.old/usr/bin/srmrm osg-wn-client/usr/bin/srmrm
--- osg-wn-client.old/usr/bin/srmrm	2013-06-08 04:24:41.000000000 +0000
+++ osg-wn-client/usr/bin/srmrm	2013-08-12 23:42:54.797501350 +0000
@@ -1,7 +1,7 @@
 #!/bin/sh
 
-if [ -e /etc/sysconfig/dcache-srmclient ]; then
-    . /etc/sysconfig/dcache-srmclient
+if [ -e ${OSG_LOCATION-}/etc/sysconfig/dcache-srmclient ]; then
+    . ${OSG_LOCATION-}/etc/sysconfig/dcache-srmclient
 fi
 
 if [ -z "$SRM_PATH" ]; then
diff -Naur --exclude=.svn --exclude=CVS --exclude=.git --exclude=_darcs --exclude=.hg osg-wn-client.old/usr/bin/srmrmdir osg-wn-client/usr/bin/srmrmdir
--- osg-wn-client.old/usr/bin/srmrmdir	2013-06-08 04:24:41.000000000 +0000
+++ osg-wn-client/usr/bin/srmrmdir	2013-08-12 23:43:02.940492768 +0000
@@ -1,7 +1,7 @@
 #!/bin/sh
 
-if [ -e /etc/sysconfig/dcache-srmclient ]; then
-    . /etc/sysconfig/dcache-srmclient
+if [ -e ${OSG_LOCATION-}/etc/sysconfig/dcache-srmclient ]; then
+    . ${OSG_LOCATION-}/etc/sysconfig/dcache-srmclient
 fi
 
 if [ -z "$SRM_PATH" ]; then
diff -Naur --exclude=.svn --exclude=CVS --exclude=.git --exclude=_darcs --exclude=.hg osg-wn-client.old/usr/bin/srmstage osg-wn-client/usr/bin/srmstage
--- osg-wn-client.old/usr/bin/srmstage	2013-06-08 04:24:41.000000000 +0000
+++ osg-wn-client/usr/bin/srmstage	2013-08-12 23:43:05.740495125 +0000
@@ -1,7 +1,7 @@
 #!/bin/sh
 
-if [ -e /etc/sysconfig/dcache-srmclient ]; then
-    . /etc/sysconfig/dcache-srmclient
+if [ -e ${OSG_LOCATION-}/etc/sysconfig/dcache-srmclient ]; then
+    . ${OSG_LOCATION-}/etc/sysconfig/dcache-srmclient
 fi
 
 if [ -z "$SRM_PATH" ]; then
